Makefile.hoot

```diff
@@ -399,11 +399,9 @@ else
 	echo "UI tests must specify both --with-services and --with-uitests."
 endif
 
-HOOT_VERSION_FILE: hoot-core/src/main/cpp/hoot/core/info/VersionDefines.h
-  -include HOOT_VERSION_FILE
-
-hoot-core/src/main/cpp/hoot/core/info/VersionDefines.h:
```

Comment:
@bmarchant I made this change because we need `HOOT_VERSION_GEN` to always run.  I had whacked a `FORCE` target that I didn't consider the impact of here https://github.com/ngageoint/hootenanny/commit/cbbaa64860f9e61521d7468a8c582284692a0c49 that I think necessitated your change that I'm now fixing with `.PHONY`.

For some reason the `make archive` step was failing in some iterations where I tried to restore `FORCE` and in other iterations it was leading to an infinite loop calling `HOOT_VERSION_GEN`.

This configuration seems to work in forcing the regeneration of VersionDefines.h on every build without the above defect.

Reply:
Will regenerating `VersionDefines.h` every time `make` is run cause a major rebuild?  If so, this would slow down compilation during development of Hootenanny outside of docker containers.  Maybe somewhere in the docker container the change could be made using `sed` or something like that.

Reply:
I think with the older `FORCE` target this should have also been the case.  But the `HOOT_VERSION_GEN` script only regenerates `VersionDefines.h` if the version has changed so I think it should be ok.  If it has changed that would necessitate a core rebuild of the three cpp classes that include it.